internal/models/reservation_model.go

```diff
@@ -48,8 +48,16 @@ type AWSReservation struct {
 	InstanceType string `db:"instance_type" json:"instance_type"`
 
 	// Amount of instances to provision of type: Instance type.
-	Amount int64 `db:"amount" json:"amount"`
+	Amount int32 `db:"amount" json:"amount"`
 
 	// The ID of the image from which the instance is created.
-	ImageID int64 `db:"image_id" json:"image_id"`
+	ImageID string `db:"image_id" json:"image_id"`
+}
```

Comment:
Please update the comment so it's obvious this is an AMI not IB ID.

Reply:
I thought of changing it to AMI, What do you think?

Reply:
Whatever you like, the argument in the AWS SDK is called ImageID actually so both are valid.

Reply:
If you still want to hear my opinion, I would go for the rename, AMI is definitely widely popular term and it is crystal clear for everyone.